cmake/os_flags.cmake

```diff
@@ -167,6 +167,21 @@ macro(ie_add_compiler_flags)
     endforeach()
 endmacro()
 
+#
+# Forced includes certain header file to all target source files
+#
+function(ie_force_include target scope header_file)
```

Comment:
Why do we need this function? Can we use only `ov_force_include`?

Reply:
All IE macros start with ie_ but in common libraries macros looks like ov_*. We can use ov_force_include, but it looks ugly inside IE. It might be better to move ov_force_include to CmakeLists.txt for common libraries.

Reply:
If we move `ov_force_include ` to openvino folder it is not solve the issue with the code duplication.
I suppose that we can rename global for OpenVINO functions to `ov_` and use it in all OpenVINO components. IE specific functions will have the prefix `ie_`

@ilya-lavrenov What do you think?

Reply:
I'm OK to rename

Reply:
Ok, renamed